src/views/SystemIntake/ContactDetails/index.tsx

```diff
@@ -68,39 +70,43 @@ type ContactDetailsProps = {
 };
 
 const ContactDetails = ({ systemIntake }: ContactDetailsProps) => {
-  const {
-    id,
-    requestType,
-    requester,
-    businessOwner,
-    productManager,
-    isso,
-    governanceTeams
-  } = systemIntake;
+  const { id, requestType, requester, governanceTeams } = systemIntake;
   const formikRef = useRef<FormikProps<ContactDetailsForm>>(null);
   const { t } = useTranslation('intake');
   const history = useHistory();
+
+  const flags = useFlags();
+
   const [isReqAndBusOwnerSame, setReqAndBusOwnerSame] = useState(false);
```

Comment:
This state should probably use a hook to compare the requester and business owner, once loaded.  When loading the page, the checkbox should be checked by default if the two are the same.

```
 useEffect(() => {
    if (intakeRequester.euaUserId === contacts?.businessOwner.euaUserId) {
      setReqAndBusOwnerSame(true);
    }
  }, [contacts, intakeRequester]);

```
This should be the same for productOwner

In the JSX:
```
checked={isReqAndBusOwnerSame}
onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
                      setReqAndBusOwnerSame(!isReqAndBusOwnerSame);
                      if (e.target.checked) {
                        setFieldValue(
                          'businessOwner.component',
                          values.requester.component
                        );
                      }
                    }}
```

Because this data isn't being persisted in the form mutation, it's fine to have it's own state, otherwise this could be a more default checkbox Formik component